perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/pages/AppsView.ui.xml

```diff
@@ -15,7 +15,15 @@
 		<b:Heading size="H2" ui:field="pageTitle">
 			<b:Icon ui:field="icon" type="ARCHIVE" fixedWidth="true" />
 			<b.html:Text text="My registrations" ui:field="text"/>
-			<b.html:Small text=""/>
+			<b.html:Small text="" marginRight="20"/>
+			<b:ButtonGroup>
+				<p:PerunButton icon="FILTER" type="PRIMARY" ui:field="filterButton">Filter</p:PerunButton>
+				<b:Button dataToggle="DROPDOWN" type="PRIMARY" ui:field="dropdown"/>
```

Comment:
The white button maybe looks better for me.
```suggestion
				<p:PerunButton icon="FILTER" ui:field="filterButton">Filter</p:PerunButton>
				<b:Button dataToggle="DROPDOWN" ui:field="dropdown"/>
```
![image](https://user-images.githubusercontent.com/70371638/192249417-48f4a599-63f5-49e5-b2e4-f323cea320f5.png)
![image](https://user-images.githubusercontent.com/70371638/192249490-6abf8d28-4302-4c9a-8b48-2fcd6d01ee82.png)